daq_lib.py

```diff
@@ -740,12 +740,12 @@ def detectorArm(angle_start,image_width,number_of_images,exposure_period,filepre
   logger.info("\narm time = " + str(armTime) +"\n")
   return
 
-def checkC2C_X(x,fovx,unitscaling): # this is to make sure the user doesn't make too much of an x-move in C2C
+def checkC2C_X(x,fovx): # this is to make sure the user doesn't make too much of an x-move in C2C
   scalePixX = getPvDesc("image_X_scalePix")
   centerPixX = getPvDesc("image_Y_centerPix")
   xpos = beamline_lib.motorPosFromDescriptor("sampleX")
   target = xpos + ((x-centerPixX) * (fovx/scalePixX))
-  target = unitscaling * target
+  target = target * daq_utils.unitScaling  # unitScaling multiplier used to adjust units between microns and mm
```

Comment:
using the value directly is a good change! you have prevented changing the function call signature, which means you don't have to check other references to it